Smart Village App/views/calendar/entry/edit-participation.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2017 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+use humhub\modules\calendar\models\forms\CalendarEntryParticipationForm;
+use humhub\modules\content\widgets\richtext\RichTextField;
+use humhub\modules\ui\form\widgets\ActiveForm;
+
+/* @var $form ActiveForm */
+/* @var $calendarEntryParticipationForm CalendarEntryParticipationForm */
+
+$entry = $calendarEntryParticipationForm->entry;
+?>
+<div class="modal-body">
+    <?= $form->field($entry, 'participation_mode')
+        ->dropDownList(CalendarEntryParticipationForm::getModeItems(), ['data-action-change' => 'changeParticipationMode']) ?>
+
+    <div class="participationOnly" style="<?= $entry->participation->isEnabled() ? '' : 'display:none' ?>">
+        <?= $form->field($entry, 'max_participants')->textInput() ?>
+
+        <?= $form->field($entry, 'participant_info')->widget(RichTextField::class, ['placeholder' => Yii::t('CalendarModule.base', 'Additional Information for participants'), 'pluginOptions' => ['maxHeight' => '300px']])->label(false) ?>
+
+        <?php if (!$entry->isNewRecord) : ?>
+            <?= $form->field($calendarEntryParticipationForm, 'sendUpdateNotification')->checkbox() ?>
+        <?php endif; ?>
+    </div>
+</div>
```

Comment:
So this file overrides the original one, right? can you point out the difference between them or maybe link to the original one somewhere so we can have a better comparison. With just the raw new file it is hard to review.

Reply:
there is an error sadly when having it on staging:

![Bildschirmfoto 2022-07-07 um 12 09 57](https://user-images.githubusercontent.com/1942953/177749449-9dc3b269-639a-4655-9de3-7994ede19605.png)

can you understand whats the problem?

Reply:
@gaurav-handysolver does the variable somehow need to be defined differently than `/* @var $calendarEntryParticipationForm CalendarEntryParticipationForm */`? it is somehow a magic comment right now?

> yii\base\ErrorException: Undefined variable: calendarEntryParticipationForm in /var/www/localhost/htdocs/themes/Smart Village App/views/calendar/entry/edit-participation.php:15

Reply:
Hi @donni106 
Where you place theme folder(Smart Village App)? Can you send the whole project directory like this

![image](https://user-images.githubusercontent.com/103625795/177765320-540ac989-b61f-4309-9493-d591b1edc803.png)

Reply:
![Bildschirmfoto 2022-07-07 um 13 52 07](https://user-images.githubusercontent.com/1942953/177766868-2da4bb3e-0a58-4d6e-9c1f-92cb7c354964.png)

which worked, as already sent in the first pull request:

![Bildschirmfoto 2022-07-06 um 15 49 46](https://user-images.githubusercontent.com/1942953/177566260-5af719f1-eb8b-4d5b-9509-b0b1e9781071.png)

hmmm...so something docker related?

Reply:
on the server we have:

![Bildschirmfoto 2022-07-07 um 14 08 51](https://user-images.githubusercontent.com/1942953/177769737-aa04e16b-9723-43c4-8149-064da6819434.png)

and inside themes:

![Bildschirmfoto 2022-07-07 um 14 09 04](https://user-images.githubusercontent.com/1942953/177769770-9971aa42-87bd-440d-9ae8-dc83cb96f5df.png)

Reply:
Hi @donni106 
Then it might be docker related problem.

Reply:
Ok, so we have identical folder structures but different outcomes. Let's think of what could possibly go wrong.

Where does the definition of `calendarEntryParticipationForm` happen?

Reply:
I only see `CalendarEntryForm.php` in `modules->calendar->models->forms`:

![Bildschirmfoto 2022-07-07 um 23 44 47](https://user-images.githubusercontent.com/1942953/177876729-60e8a99c-fb2b-4305-8aca-72fe6f6e83aa.png)

but the checkboxes are there:

![175336331-1ea8a93f-8712-4ad4-b0b4-b6fe6936086b](https://user-images.githubusercontent.com/1942953/177278195-f6493e89-0274-4e1f-b04d-64c8a9d57edf.png)

maybe its a difference between our 1.9.3 and your version?

> share the docker configuration

what exactly can i share with you that would help here? some files or file contents?

Reply:
Hi @donni106 
I think it is calendar module version problem. what version do you using ?
we are using 1.2.2 version of calendar module.

![image](https://user-images.githubusercontent.com/103625795/177954513-66e7b6d1-15bd-412e-8382-3a653b511cb2.png)

You can check version number in module->calendar->module.json file.

Reply:
Hi @donni106 
Calendar is an external humhub module so, if we downgrade our humhub version to 1.9.3 then it doesn't affect. and we also don't get the 1.11.1 version of calendar module.

Reply:
Hi @donni106 
We setup your humhub version (1.9.3) and also calendar module (1.1.11) and yes, "calendar-entry-form" file is not present in calendar module.

I think this might be happen because of calendar module version.

Now, one thing we can do is make another branch issue-1A and push a commit so, that this issue will be resolved.
Thanks!

Reply:
sounds good, but why `issue-1A`? you can update the `issue-2` branch accordingly i would suggest